osquery/bro/BrokerManager.cpp

```diff
@@ -0,0 +1,309 @@
+#include <osquery/system.h>
+#include <osquery/flags.h>
+#include <osquery/logger.h>
+#include <osquery/sql.h>
+
+
+#include <broker/broker.hh>
+#include <broker/endpoint.hh>
+#include <broker/message_queue.hh>
+
+#include "osquery/bro/utils.h"
+
+#include "osquery/bro/BrokerManager.h"
+#include "osquery/bro/QueryManager.h"
+#include <iostream>
+#include <sstream>
+#include <list>
+#include <stdlib.h>     /* srand, rand */
+#include <time.h>
+
+namespace osquery {
+
+
+    BrokerManager *BrokerManager::_instance = nullptr;
```

Comment:
this is necessary to initialize the kInstance_ of Singletons if they are stored as pointers. The Singleton can only be retrieved by Class::getInstance()